plugins/generic/customBlockManager/CustomBlockPlugin.inc.php

```diff
@@ -176,13 +183,29 @@ function manage($verb, $args) {
 	 */
 	function getContents(&$templateMgr) {
 		$journal =& Request::getJournal();
+		$journalId = $journal->getId();
+		
 		if (!$journal) return '';
+		
+		$locale = AppLocale::getLocale();
+		$primaryLocale = AppLocale::getPrimaryLocale();
+		$blockContent = $this->getSetting($journalId, 'blockContent');
+		$blockContentTxt = '';
+	
+		if (!is_array($blockContent)) {
+			$this->updateSetting($journalId, 'blockContent', array($primaryLocale => $blockContent));
+			$blockContentTxt = $blockContent;
+		}
```

Comment:
Generally we use...
    } else {
...rather than splitting onto two lines. Just a minor coding convention suggestion.